arpa-exporter/tests/conftest.py

```diff
@@ -7,6 +7,7 @@
 import moto.ses.models
 import pytest
 
+os.environ.setdefault("DD_TRACE_ENABLED", "false")
```

Comment:
Also unrelated to the bug. I just noticed that `ddtrace` was logging output (complaining that it couldn't reach Datadog) during unit test execution. This disables `ddtrace` unless it has been explicitly enabled (which may be the case if Datadog's testing suite is in-use, e.g. in a CI environment).